guides/2022-04-05-guide-to-conduit-connector-lifecycle.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "The Guide to the Conduit Connector Lifecyle"
+slug: guide-to-conduit-connector-lifecycle
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['File Connector']
+description: Learn how to build a Conduit Algolia Connector
+keywords: ['data', 'connector', 'algolia', 'conduit']
+---
+
+# The Guide to the Conduit Connector Lifecyle
+
+The goal of Conduit is to have the best experience when it comes to building connectors. The Conduit Connector Lifecyle describes TODO. 
+
+Listed in the order they are called, here are the methods you can implement for Conduit Connectors:
+
+- Configure
+- Open
+- Read
+- Ack
+- Write Async
+- Write
+- Flush
+- Teardown
```

Comment:
We should definitely split these for source and destination connectors. Also I'm not sure I would even mention Write Async in this list because it makes things confusing, I would maybe add an asterisk behind "Write" and a footnote below the list that you can choose to implement "Write Async" instead, but you shouldn't do that unless you know exactly what you're doing. Given that this is a beginner's guide they should probably use Write at this point.

`Configure`, `Open` and `Teardown` are the same for a source or destination connector, so we could describe those together, but the rest should probably be grouped under each connector type.